pkg/security/metrics/metrics.go

```diff
@@ -101,6 +101,9 @@ var (
 	// MetricPerfBufferBytesRead is the name of the metric used to count the number of bytes read from a perf buffer
 	// Tags: map
 	MetricPerfBufferBytesRead = newRuntimeMetric(".perf_buffer.bytes.read")
+	// MetricPerfBufferBytesFree is the name of the metric used to count the number of bytes left in the ring buffer
+	// Tags: map
+	MetricPerfBufferBytesFree = newRuntimeMetric(".perf_buffer.bytes.free")
```

Comment:
We discussed offline about:
- changing this metric to a usage percentage
- tagging this metric with the discrete size of the maps